internal/docs/assets/reader.css

```
@@ -0,0 +1,136 @@
+:root {
+  --reader-body-font: "Iowan Old Style", "Palatino Linotype", Charter, Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Charter" to be "charter" (value-keyword-case)

(value-keyword-case)